openmetadata-service/src/main/java/org/openmetadata/service/util/AsciiTable.java

```diff
@@ -12,6 +12,16 @@ public record AsciiTable(
   private static final String DEFAULT_COLUMN_NAME = "(No column name)";
   private static final String DEFAULT_NO_VALUE = "-";
 
+  private static final String logo =
```

Comment:
@harshach  let me know if this needs to be removed :)

Reply:
we had another one maybe we can reuse the same?